src/database/sqlite/mod.rs

```diff
@@ -2423,6 +2432,7 @@ CREATE TABLE labels (
 
         // SqliteDb new is doing the migration.
         let db = SqliteDb::new(db_path, None, &secp).unwrap();
+        db.maybe_apply_migrations().unwrap();
```

Comment:
The comment just above is wrong "SqliteDb::new is doing the migration". There is a similar comment at the top of the previous test which needs to be updated too.

Reply:
Thanks, I think I've removed them all now. There was one place where the test avoided using `SqliteDb::new()` specifically not to do the migration, and so I changed it now to use `SqliteDb::new()` and removed the comment.